beetsplug/lyrics.py

```diff
@@ -483,7 +483,10 @@ def extract_lyrics(self, html):
         if not soup:
             return None
 
-        return soup.find("div", class_="song-text").get_text()
+        c = soup.find("div", class_="song-text")
```

Comment:
Small nitpick: Better rename this variable to something meaningful, e.g. `lyrics_div` (which is a name also used by another lyrics source).